packages/zudoku/src/lib/components/navigation/SidebarCategory.tsx

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-console */
```

Comment:
Is this a leftover we don't need anymore?

Reply:
nyah, this is a miss out on my part, removing the line!